python/lsst/atmospec/utils.py

```diff
@@ -565,15 +607,38 @@ def getFilterAndDisperserFromExp(exp):
     Returns
     -------
     filter, disperser : `tuple` of `str`
-        The filter and the disperser names, as strings.
+        The filter and the disperser names.
     """
-    filterFullName = exp.getFilter().physicalLabel
-    if FILTER_DELIMITER not in filterFullName:
-        filt = filterFullName
-        grating = exp.getInfo().getMetadata()['GRATING']
-    else:
-        filt, grating = filterFullName.split(FILTER_DELIMITER)
-    return filt, grating
+    filterName = exp.getFilter().physicalLabel
+    return getFilterAndDisperserFromFilterName(filterName)
+
+
+def parseFilterName(filterName):
+    """Parse the filter name.
+
+    Parameters
+    ----------
+    filterName : `str`
+        The filter name.
+
+    Returns
+    -------
+    `dict`
```

Comment:
I think this line needs to read something like
```
wheelConfiguration : `dict`
```
so that it has a variable name, not just a type.